tests/functional/strategy/test_config.py

```diff
@@ -45,6 +46,7 @@ def test_strategy_setEmergencyExit(strategy, gov, strategist, rando, chain):
     "getter,setter,val",
     [
         ("strategist", "setStrategist", None),
+        ("rewards", "setRewards", None),
```

Comment:
Shit, the way this test works, you should skip governance trying to set it

Reply:
Fixed by adding `caller` into parametrize setup.